model/category.go

```diff
@@ -8,11 +8,12 @@ import "fmt"
 
 // Category represents a feed category.
 type Category struct {
-	ID          int64  `json:"id"`
-	Title       string `json:"title"`
-	UserID      int64  `json:"user_id"`
-	FeedCount   int    `json:"-"`
-	TotalUnread int    `json:"-"`
+	ID           int64  `json:"id"`
+	Title        string `json:"title"`
+	UserID       int64  `json:"user_id"`
+	HideGlobally bool   `json:"hide_globally"`
```

Comment:
it's a small details but I would keep naming consistent.

The column in the database is `hide_global_unread`, but the JSON field is `hide_globally`, and the struct field is `HideGlobally`.

Reply:
renamed the db field to `hide_globally` to match